satellite/rewards/partners.go

```diff
@@ -13,105 +15,144 @@ var DefaultPartnersDB = func() PartnersDB {
 	return db
 }()
 
+func parseUUID(s string) *uuid.UUID {
```

Comment:
Can you add a comment to this function that it should only be used for hardcoded UUIDs?

Reply:
but this function is independent from hardcoded UUIDs. It just parses a string to UUID.

Reply:
Yes, I know but it panics if the UUID is not valid, so we should use it only when we control what UUID we are converting. I would only like to panic if it's a bug passing one of the UUIDs of the static list but I wouldn't panic if the UUID could come from other sources like user input, data store, etc.

Reply:
make sense, I will add comment